README.md

```diff
@@ -45,17 +45,17 @@ in your Spark applications.
 The connector project has several branches, each of which map into different
 supported versions of  Spark and Cassandra. For previous releases the branch is
 named "bX.Y" where X.Y is the major+minor version; for example the "b1.6" branch
-corresponds to the 1.6 release. The "master" branch will normally contain
+corresponds to the 1.6 release. The "trunk" branch will normally contain
 development for the next connector release in progress.
 
 Currently, the following branches are actively supported: 
-3.5.x ([master](https://github.com/datastax/spark-cassandra-connector/tree/master)),
-3.4.x ([b3.4](https://github.com/datastax/spark-cassandra-connector/tree/b3.4)),
-3.3.x ([b3.2](https://github.com/datastax/spark-cassandra-connector/tree/b3.3)),
-3.2.x ([b3.2](https://github.com/datastax/spark-cassandra-connector/tree/b3.2)),
-3.1.x ([b3.1](https://github.com/datastax/spark-cassandra-connector/tree/b3.1)),
-3.0.x ([b3.0](https://github.com/datastax/spark-cassandra-connector/tree/b3.0)) and 
-2.5.x ([b2.5](https://github.com/datastax/spark-cassandra-connector/tree/b2.5)).
+3.5.x ([trunk](https://github.com/apache/cassandra-spark-connector/tree/trunk)),
```

Comment:
We should really have a discussion about the number of branches we want to support given the move from Datastax to Apache - this seems excessive and maybe unnecessary, but I'm not sure why we currently have this many (Spark and Scala are sometimes difficult from a compatibility perspective... that said, looking at the compatibility chart, it seems like we may be able to reduce this down to 3.5.1 and 2.5 if 3.5.1 works on older Spark versions (back to 3.0, which most libraries _should_ be able to do I think).

Reply:
agree.   we're free to strip this down to whatever we think is maintainable, and that could well be just one non-trunk version.